ledger/src/blockstore_processor.rs

```diff
@@ -2884,6 +2886,155 @@ pub mod tests {
         }
     }
 
+    #[test]
+    fn test_transaction_result_influences_blockhash() {
```

Comment:
Should be `test_transaction_result_does_not_affect_bankhash()`

Reply:
Done